src/Components/ServiceScene/LogsVolumePanel.tsx

```diff
@@ -44,68 +41,19 @@ export class LogsVolumePanel extends SceneObjectBase<LogsVolumePanelState> {
             .setTitle('Log volume')
             .setOption('legend', { showLegend: false })
             .setData(
-              new SceneDataTransformer({
-                $data: new SceneQueryRunner({
-                  datasource: explorationDS,
-                  queries: [
-                    buildLokiQuery(
-                      `sum by (level) (count_over_time(${LOG_STREAM_SELECTOR_EXPR} | drop __error__ [$__auto]))`,
-                      { legendFormat: '{{level}}' }
-                    ),
-                  ],
-                }),
-                transformations: [
-                  () => (source: Observable<DataFrame[]>) => {
-                    return source.pipe(
-                      map((data: DataFrame[]) => {
-                        return data.sort((a, b) => {
-                          const aName = a.fields[1].config.displayNameFromDS;
-                          const aVal = aName?.includes('error')
-                            ? 4
-                            : aName?.includes('warn')
-                            ? 3
-                            : aName?.includes('info')
-                            ? 2
-                            : 1;
-                          const bName = b.fields[1].config.displayNameFromDS;
-                          const bVal = bName?.includes('error')
-                            ? 4
-                            : bName?.includes('warn')
-                            ? 3
-                            : bName?.includes('info')
-                            ? 2
-                            : 1;
-                          return aVal - bVal;
-                        });
-                      })
-                    );
-                  },
-                ],
-              })
+              getQueryRunner(
+                buildLokiQuery(
+                  `sum by (detected_level) (count_over_time(${LOG_STREAM_SELECTOR_EXPR} | drop __error__ [$__auto]))`,
```

Comment:
```suggestion
                  `sum by (LEVEL_VARIABLE_VALUE) (count_over_time(${LOG_STREAM_SELECTOR_EXPR} | drop __error__ [$__auto]))`,
```

Reply:
Fixed in 3f8eaa1daea4926562327d43ebb174e40129eab8